test/integration/visitor_can_edit_cart_test.rb

```diff
@@ -88,7 +88,7 @@ class VisitorCanEditCartTest < ActionDispatch::IntegrationTest
       within ('.quantity') do
         assert page.has_content?("1")
       end
-      within ('.price') do
+      within ('.subtotal') do
```

Comment:
`(...)` interpreted as grouped expression.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.